src/core/public/application/ui/app_not_found_screen.tsx

```diff
@@ -28,10 +28,14 @@
  * under the License.
  */
 
-import { EuiEmptyPrompt, EuiPage, EuiPageBody, EuiPageContent } from '@elastic/eui';
+import { EuiButton, EuiEmptyPrompt, EuiPage, EuiPageBody, EuiPageContent } from '@elastic/eui';
 import React from 'react';
 import { FormattedMessage } from '@osd/i18n/react';
 
+const goToHomePage = () => {
+  window.location.href = '/';
```

Comment:
This will work with no base path set but prevents something like:
```
{OSD_URL}/_dashboards/app
```
from working.

What would want to do is something like:
```tsx
 const baseUrl = window.location.href.split('/app')[0] || '';
 window.location.href = baseUrl;
 ```